### PR TITLE
Added second classes for tracker and keyword so they are easier to find

### PR DESCRIPTION
**Pull request by benfrancis314**
Added a secondary, static class for the tracker and keyword so they can be found without determining the current user settings.

### CODE REVIEW

File: extension/content_script.js [hunk 1]
```diff
@@ -132,9 +131,6 @@ function calculateTrackerLife() {
 	const speed_bias_ms = 500; // Half of a second; is this too long?
 
 	let sentenceId = tracker.getSentenceId();
-	let sentencePtr = doc.getSentence(sentenceId);
-
-	let containerId = sentencePtr.containerId;
```
Comment (benfrancis314) [inline extension/content_script.js:142]: I believe these are not used in this function

File: extension/content_script.js [hunk 2]
```diff
@@ -24,7 +24,6 @@ var timer = null;
 let speed = null; // WPM, Base speed, not accounting for sentence length; adjustable w/ D/S
 // Persistent settings.
 let settings = window.settings;
-var speed_bias = 500; // Minimum amount of speed spent on each sentence (in milliseconds)
```
Comment (benfrancis314) [inline extension/content_script.js:32]: Not used as global variable; used as constant w/in function that needs it

File: extension/content_script.js [hunk 3]
```diff
@@ -183,19 +179,10 @@ function moveOne(dir) { // Sets start and end
...
 function scrollUp() {
-	// Check to see if we should use old style or look for new
-	let tracker_style_current = null; // Depends on if style was just switched
-	if ($("."+trackerStyle)) {
-		tracker_style_current = trackerStyle
-	} else { 
-		let displaySettings = display.getSettings();
-		tracker_style_current = "trackerHighlighter"+displaySettings[1]+"Shadow"+displaySettings[2]; 
-	}
-
```
Comment (benfrancis314) [inline extension/content_script.js]: No longer need to check for this; just need to look for "trackerClass", which is static

File: extension/content_script.js [hunk 4]
```diff
@@ -212,21 +199,12 @@ function scrollUp() {
...
 function scrollDown() {
-	// Check to see if we should use old style or look for new
-	let tracker_style_current = null; // Depends on if style was just switched
-	if ($("."+trackerStyle)) {
-		tracker_style_current = trackerStyle
-	} else { 
-		let displaySettings = display.getSettings();
-		tracker_style_current = "trackerHighlighter"+displaySettings[1]+"Shadow"+displaySettings[2]; 
-	}
-
```
Comment (benfrancis314) [inline extension/content_script.js]: Same as scrollUp

File: extension/content_script.js (hunk 1 above)
Comment (sjonany) [reply extension/content_script.js:142]: Nice, thanks for catching this!

File: extension/content_script.js [hunk 5]
```diff
@@ -314,27 +297,26 @@ function highlightKeyWords(container, start, end) {
...
 
 /*
 Fade the current tracker indicator according to the calculated speed.
 */
 function fadeTracker() {
-	let displaySettings = display.getSettings();
-	let keywordStyle = "keyWord"+displaySettings[0];
 	fadeElement($("mark"));
-	fadeElement($("." + keywordStyle));
+	fadeElement($(".keywordClass"));
```
Comment (sjonany) [inline extension/content_script.js]: - please turn "keywordClass" to a variable.
- and / or create a function to get the keyword / tracker els. If someday want to change the css class, then just swap out one place.

**Review by sjonany — CHANGES_REQUESTED** [review]

File: extension/content_script.js (hunk 5 above)
Comment (benfrancis314) [reply extension/content_script.js]: Oh nice yep agree

File: extension/content_script.js [hunk 6]
```diff
@@ -9,6 +9,9 @@ if (window[namespace] === true) {
 	window[namespace] = true;
 }
 
+const keywordClass = "keywordClass" // Name of class for FINDING keywords (no styling)
+const trackerClass = "trackerClass" // Name of class for FINDING the tracker (no styling)
```
Comment (benfrancis314) [inline extension/content_script.js:16]: Changes here, and then waterfalled down to all usages of keywordClass and trackerClass below

**Comment by sjonany** [comment]
Very nice! LGTM

**Review by sjonany — APPROVED** [review]

**Pull request merged**